pages/common/snowsql.md

```diff
@@ -0,0 +1,24 @@
+# snowsql
+
+> SnowSQL command-line client for Snowflake's Data Cloud.
+> More information: <https://docs.snowflake.com/en/user-guide/snowsql.html>.
+
+- Connect to a specific instance at account.snowflakecomputing.com (password can be provided in prompt or configuration file):
+
+`snowsql --accountname {{account}} --username {{username}} --dbname {{database}} --schemaname {{schema}}`
+
+- Connect to an instance specified by a specific configuration file (by default, `~/.snowsql/config`):
+
+`snowsql --config {{~/.snowsql/config}}`
```

Comment:
If we are using a non-default file:
```suggestion
`snowsql --config {{path/to/configuration_file}}`
```
Also, we try to avoid abbreviations

Reply:
Done, also changed in the translation.